examples/meson.build

```diff
@@ -149,6 +149,14 @@ clients = {
 		'dep': wlroots,
 		'proto': ['wlr-virtual-pointer-unstable-v1'],
 	},
+	'input-method-keyboard-grab': {
+		'src': 'input-method-keyboard-grab.c',
+		'dep': xkbcommon,
+		'proto': [
+			'input-method-unstable-v2',
+			'text-input-unstable-v3',
```

Comment:
`text-input-unstable-v3` shouldn't be required.

Reply:
Removed, done.